### PR TITLE
review components page and related overview pages

### PR DESCRIPTION
**Pull request by mbuccarello**
Hello, I did a review of the components page, keep in mind in version 1.18 there are a lot of pages introduced into the glossary page.

### CODE REVIEW

File: content/it/docs/concepts/overview/components.md [hunk 1]
```diff
@@ -53,27 +53,25 @@ Alcuni esempi di controller gestiti dal kube-controller-manager sono:
...
 ### cloud-controller-manager
 
-Il [cloud-controller-manager](/docs/tasks/administer-cluster/running-cloud-controller/) esegue i controller che interagiscono con i cloud provider responsabili per la gestione dell'infrastruttura sottostante al cluster, in caso di deployment in cloud.
-Il cloud-controller-manager è una funzionalità alpha introdotta in Kubernetes 1.6.
+{{< glossary_definition term_id="cloud-controller-manager" length="short" >}}
 
-Il cloud-controller-manager esegue esclusivamente i cicli di controllo specifici dei cloud provider.
-È possibile disabilitare questi cicli di controllo usando il kube-controller-manager.
-È inoltre possibile disabilitare i cicli di controllo settando il parametro `--cloud-provider` con il valore `external` durante l'esecuzione del kube-controller-manager.
+Il cloud-controller-manager esegue controller specifiche del tuo cloud provider.
```
Comment (Fale) [inline content/it/docs/concepts/overview/components.md:58]: I would say that controller is to be considered as male not female. What's your feeling on this?

File: content/it/docs/concepts/overview/components.md [hunk 2]
```diff
@@ -53,27 +53,25 @@ Alcuni esempi di controller gestiti dal kube-controller-manager sono:
...
+Se hai una installazione Kubernetes on premises, o un ambiente di laboratorio
```
Comment (Fale) [inline content/it/docs/concepts/overview/components.md:59]: I would put a "di" before "Kubernetes". Also, "on-premises" instead of "on premises", I think is more clear

File: content/it/docs/concepts/overview/components.md [hunk 3]
```diff
@@ -53,27 +53,25 @@ Alcuni esempi di controller gestiti dal kube-controller-manager sono:
...
+nel tuo PC, il cluster non ha un cloud controller manager.
```
Comment (Fale) [inline content/it/docs/concepts/overview/components.md:60]: "cloud-controller-manager" is a name all together, so I think we should substitute "cloud controller manager" with "cloud-controller-manager"

File: content/it/docs/concepts/overview/components.md [hunk 4]
```diff
@@ -53,27 +53,25 @@ Alcuni esempi di controller gestiti dal kube-controller-manager sono:
...
 
-Il cloud-controller-manager permette l'evoluzione indipendente al codice di Kubernetes e a quello dei singoli cloud vendor.
-Precedentemente, il codice core di Kubernetes dipendeva da implementazioni specifiche dei cloud provider.
-In futuro, implementazioni specifiche per singoli cloud provider devono essere mantenuti dai cloud provider interessati e collegati al cloud-controller-manager.
+Come nel kube-controller-manager, il cloud-controller-manager combina diversi control loops
```
Comment (Fale) [inline content/it/docs/concepts/overview/components.md:62]: I think "control loops" is not to be considered as a specific term, so I would translate it. What about "cicli di controllo" or "sistemi di controllo"?

File: content/it/docs/concepts/overview/components.md [hunk 5]
```diff
@@ -53,27 +53,25 @@ Alcuni esempi di controller gestiti dal kube-controller-manager sono:
...
-È inoltre possibile disabilitare i cicli di controllo settando il parametro `--cloud-provider` con il valore `external` durante l'esecuzione del kube-controller-manager.
+Il cloud-controller-manager esegue controller specifiche del tuo cloud provider.
+Se hai una installazione Kubernetes on premises, o un ambiente di laboratorio
+nel tuo PC, il cluster non ha un cloud controller manager.
 
-Il cloud-controller-manager permette l'evoluzione indipendente al codice di Kubernetes e a quello dei singoli cloud vendor.
-Precedentemente, il codice core di Kubernetes dipendeva da implementazioni specifiche dei cloud provider.
-In futuro, implementazioni specifiche per singoli cloud provider devono essere mantenuti dai cloud provider interessati e collegati al cloud-controller-manager.
+Come nel kube-controller-manager, il cloud-controller-manager combina diversi control loops 
+logicamente indipendenti in un singolo binario che tu puoi eseguire come un singolo processo. Tu puoi
```
Comment (Fale) [inline content/it/docs/concepts/overview/components.md:63]: I would remove the two "tu". They feel "redundant" in Italian in my opinion

File: content/it/docs/concepts/overview/components.md [hunk 6]
```diff
@@ -53,27 +53,25 @@ Alcuni esempi di controller gestiti dal kube-controller-manager sono:
...
+scalare orizzontalmente ( eseguire più di una copia ) per migliorare la responsività o per migliorare la tolleranza ai fallimenti.
```
Comment (Fale) [inline content/it/docs/concepts/overview/components.md:64]: there should not be a space between a punctuation symbol and it's related word. So: "( eseguire più di una copia )" "(eseguire più di una copia)"

File: content/it/docs/concepts/overview/components.md [hunk 7]
```diff
@@ -53,27 +53,25 @@ Alcuni esempi di controller gestiti dal kube-controller-manager sono:
...
 
 I seguenti controller hanno dipendenze verso implementazioni di specifici cloud provider:
 
   * Node Controller: Per controllare se sul cloud provider i nodi che hanno smesso di rispondere sono stati cancellati
-  * Route Controller: Per configurare le regole di route nella sottostante infrastruttura cloud
-  * Service Controller: Per creare, aggiornare ed eliminare i load balancer nella infrastruttura del cloud provider
-  * Volume Controller: Per creare, associare e montare i volumi e per interagire con il cloud provider per orchestrare i volumi
-
+  * Route Controller: Per configurare le rotte nella sottostante infrastruttura cloud
```
Comment (Fale) [inline content/it/docs/concepts/overview/components.md:69]: I think "rotte" is not very clear in Italian. Though, I'm not really sure how it could be written in a more clear way

File: content/it/docs/concepts/overview/components.md [hunk 8]
```diff
@@ -53,27 +53,25 @@ Alcuni esempi di controller gestiti dal kube-controller-manager sono:
...
+  * Service Controller: Per creare, aggiornare ed eliminare i load balancer del cloud provider
+ 
 ## Componenti dei Nodi
 
-I componenti di Kubernetes che girano sui Worker Node sono responsabili dell'esecuzione dei workload degli utenti.
+I componenti del nodo vengono eseguiti su ogni nodo, mantenendo i pod in esecuzione e fornendo l'ambiente di runtime Kubernetes.
```
Comment (Fale) [inline content/it/docs/concepts/overview/components.md:74]: "I componenti del nodo vengono eseguiti su ogni nodo" feels tautologic. Though, I'm not really sure how it could be written in a more clear way.

File: content/it/docs/concepts/overview/components.md [hunk 9]
```diff
@@ -89,18 +87,20 @@ I componenti di Kubernetes che girano sui Worker Node sono responsabili dell'ese
 
 ## Addons
 
-Gli Addons usano le risorse Kubernetes ({{< glossary_tooltip term_id="daemonset" >}}, {{< glossary_tooltip term_id="deployment" >}}, etc) per implementare nuove funzionalità a livello di cluster.
+Gli Addons usano le risorse Kubernetes ({{< glossary_tooltip term_id="daemonset" >}},
```
Comment (Fale) [inline content/it/docs/concepts/overview/components.md:90]: Why this change?

File: content/it/docs/concepts/overview/components.md [hunk 10]
```diff
@@ -89,18 +87,20 @@ I componenti di Kubernetes che girano sui Worker Node sono responsabili dell'ese
 
 ## Addons
 
-Gli Addons usano le risorse Kubernetes ({{< glossary_tooltip term_id="daemonset" >}}, {{< glossary_tooltip term_id="deployment" >}}, etc) per implementare nuove funzionalità a livello di cluster.
+Gli Addons usano le risorse Kubernetes ({{< glossary_tooltip term_id="daemonset" >}}, 
+{{< glossary_tooltip term_id="deployment" >}}, etc) per implementare funzionalità di cluster.
 Dal momento che gli addons forniscono funzionalità a livello di cluster, le risorse che necessitano di un namespace, vengono collocate nel namespace `kube-system`.
 
-Alcuni addons sono descritti di seguito; mentre per una più estesa lista di addons, riferirsi ad [Addons](/docs/concepts/cluster-administration/addons/).
+Alcuni addons sono descritti di seguito; mentre per una più estesa lista di addons, per favore
```
Comment (Fale) [inline content/it/docs/concepts/overview/components.md:94]: Why this change?

File: content/it/docs/reference/glossary/cloud-controller-manager.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+full_link: /docs/concepts/architecture/cloud-controller/
+short_description: >
+  Componente della control plane che integra Kubernetes con cloud providers di terze parti.
+aka: 
+tags:
+- core-object
+- architecture
+- operation
+---
+Un componente della {{< glossary_tooltip text="control plane" term_id="control-plane" >}} di kubernetes
```
Comment (Fale) [inline content/it/docs/reference/glossary/cloud-controller-manager.md:14]: Kubernetes should be capitalized ;-)

File: content/it/docs/reference/glossary/cloud-controller-manager.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+short_description: >
+  Componente della control plane che integra Kubernetes con cloud providers di terze parti.
+aka: 
+tags:
+- core-object
+- architecture
+- operation
+---
+Un componente della {{< glossary_tooltip text="control plane" term_id="control-plane" >}} di kubernetes 
+che aggiunge logiche di controllo specifiche per il cloud. Il cloud controller manager ti permette di collegare il tuo
```
Comment (Fale) [inline content/it/docs/reference/glossary/cloud-controller-manager.md:15]: s/cloud controller manager/cloud-controller-manager/

File: content/it/docs/reference/glossary/cloud-controller-manager.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+cluster con le cloud API del provider e separa le componenti che interagiscono
```
Comment (Fale) [inline content/it/docs/reference/glossary/cloud-controller-manager.md:16]: I would say "le API del cloud provider" instead of "le cloud API del provider"

File: content/it/docs/reference/glossary/cloud-controller-manager.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+con la piattaforma cloud dai componenti che interagiscono solamente col cluster.
+
+<!--more-->
+
+Disaccoppiando la logica di interoperabilità tra Kubernetes e l'infrastruttura cloud sottostante,
+il componente cloud-controller-manager abilità i cloud providers di rilasciare
```
Comment (Fale) [inline content/it/docs/reference/glossary/cloud-controller-manager.md:22]: Plural English terms brought to Italian have to be used with plural article and singular form, so it should be "i cloud provider" and not "i cloud providers"

File: content/it/docs/reference/glossary/cluster.md [hunk 15]
```diff
@@ -4,14 +4,14 @@ id: cluster
...
-   Un'insieme di macchine, chiamate nodi, che eseguono container e gestite da Kubernetes. Un cluster ha almeno un Worker Node e un Control Plane Node.
+   Un'insieme di macchine, chiamate nodi, che eseguono container gestiti da Kubernetes. Un cluster ha almeno un Worker Node e un Control Plane Node.
 
 aka: 
 tags:
 - fundamental
 - operation
 ---
-Un'insieme di macchine, chiamate nodi, che eseguono container e gestite da Kubernetes. Un cluster ha almeno un Worker Node e un Control Plane Node.
+Un'insieme di macchine, chiamate nodi, che eseguono container gestiti da Kubernetes. Un cluster ha almeno un Worker Node e un Control Plane Node.
```
Comment (Fale) [inline content/it/docs/reference/glossary/cluster.md:14]: Current English version has removed "e un Control Plane Node". I think we should match it.

File: content/it/docs/reference/glossary/cluster.md [hunk 16]
```diff
@@ -4,14 +4,14 @@ id: cluster
 date: 2019-06-15
 full_link: 
 short_description: >
-   Un'insieme di macchine, chiamate nodi, che eseguono container e gestite da Kubernetes. Un cluster ha almeno un Worker Node e un Control Plane Node.
+   Un'insieme di macchine, chiamate nodi, che eseguono container gestiti da Kubernetes. Un cluster ha almeno un Worker Node e un Control Plane Node.
```
Comment (Fale) [inline content/it/docs/reference/glossary/cluster.md:7]: Current English version has removed "e un Control Plane Node". I think we should match it.

File: content/it/docs/reference/glossary/container.md [hunk 17]
```diff
@@ -0,0 +1,19 @@
...
+full_link: /docs/concepts/overview/what-is-kubernetes/#why-containers
+short_description: >
+  Una leggera e portabile immagine eseguibile che contiene il software e tutte le sue dipendenze.
+
+aka: 
+tags:
+- fundamental
+- workload
+---
+ Una leggera e portabile immagine eseguibile che contiene il software e tutte le sue dipendenze.
```
Comment (Fale) [inline content/it/docs/reference/glossary/container.md:14]: Remove the space at the begin of the line, please

File: content/it/docs/reference/glossary/container.md [hunk 18]
```diff
@@ -0,0 +1,19 @@
+---
+title: Container
+id: container
+date: 2018-04-12
+full_link: /docs/concepts/overview/what-is-kubernetes/#why-containers
+short_description: >
+  Una leggera e portabile immagine eseguibile che contiene il software e tutte le sue dipendenze.
```
Comment (Fale) [inline content/it/docs/reference/glossary/container.md:7]: I would say "Una immagine leggera, protabile ed eseguibile che contiene un software e tutte le sue dipendenze". Also same thing for line #14 of this same file

File: content/it/docs/reference/glossary/container.md [hunk 19]
```diff
@@ -0,0 +1,19 @@
...
+aka: 
+tags:
+- fundamental
+- workload
+---
+ Una leggera e portabile immagine eseguibile che contiene il software e tutte le sue dipendenze.
+
+<!--more--> 
+
+I Containers disaccoppiano le applicazione dall'infrastruttura host sottostante e rendono semplice il deploy nei differenti cloud o sistemi operativi e anche per una semplice scalabilità
```
Comment (Fale) [inline content/it/docs/reference/glossary/container.md:18]: s/Containers/container/

File: content/it/docs/reference/glossary/control-plane.md [hunk 20]
```diff
@@ -0,0 +1,26 @@
...
+
+aka:
+tags:
+- fundamental
+---
+ Lo strato per l'orchestrazione dei container che espone le API e interfaccie per definere, deploy, e gestione del ciclo di vita dei container.
+
+ <!--more--> 
+ 
+ Questo strato è composto da diversi componenti, come ( ma non limitato a):
```
Comment (Fale) [inline content/it/docs/reference/glossary/control-plane.md:18]: Remove the space after the "("

File: content/it/docs/reference/glossary/control-plane.md [hunk 21]
```diff
@@ -0,0 +1,26 @@
...
+
+ * {{< glossary_tooltip text="etcd" term_id="etcd" >}}
+ * {{< glossary_tooltip text="API Server" term_id="kube-apiserver" >}}
+ * {{< glossary_tooltip text="Scheduler" term_id="kube-scheduler" >}}
+ * {{< glossary_tooltip text="Controller Manager" term_id="kube-controller-manager" >}}
+ * {{< glossary_tooltip text="Cloud Controller Manager" term_id="cloud-controller-manager" >}}
+
+ Questi compenti possono girare come trazionali servizi del sistema operativo (demoni) o come containers. L'host che esegue quest componenti era storicamente chiamato {{< glossary_tooltip text="masters" term_id="master" >}}.
```
Comment (Fale) [inline content/it/docs/reference/glossary/control-plane.md:26]: s/ quest / questi /
s/masters/master/

File: content/it/docs/reference/glossary/daemonset.md [hunk 22]
```diff
@@ -0,0 +1,20 @@
...
+short_description: >
+  Assicura che una copia di una Pod è attiva su tutti nodi di un cluster.
+
+aka: 
+tags:
+- fundamental
+- core-object
+- workload
+---
+ Assicura che una copia {{< glossary_tooltip text="Pod" term_id="pod" >}} è attiva su tutti nodi di un {{< glossary_tooltip text="cluster" term_id="cluster" >}}.
```
Comment (Fale) [inline content/it/docs/reference/glossary/daemonset.md:15]: s/Assicura che una copia /Assicura che una copia del /

File: content/it/docs/reference/glossary/deployment.md [hunk 23]
```diff
@@ -0,0 +1,20 @@
...
+short_description: >
+  Gestisce una applicazione replicata nel tuo cluster.
+
+aka: 
+tags:
+- fundamental
+- core-object
+- workload
+---
+ Un oggetto API che gestisce un'applicazione replicatata, tipicamente esegue Pods senza stato locale.
```
Comment (Fale) [inline content/it/docs/reference/glossary/deployment.md:15]: s/Pods/Pod/

File: content/it/docs/reference/glossary/deployment.md [hunk 24]
```diff
@@ -0,0 +1,20 @@
...
+
+<!--more--> 
+Ogni replica è rappresentata da un {{< glossary_tooltip term_id="pod" >}}, e le Pods sono distribuite attraverso i
```
Comment (Fale) [inline content/it/docs/reference/glossary/deployment.md:18]: the term "Pod" has been considered male in previous translations. So: s/e le Pods sono deistribuite/i pod sono distribuiti/

File: content/it/docs/reference/glossary/docker.md [hunk 25]
```diff
@@ -0,0 +1,17 @@
+---
+title: Docker
+id: docker
+date: 2018-04-12
+full_link: https://docs.docker.com/engine/
+short_description: >
+  Docker è una technologia software che offre una virtualizzazione a livello del sistema operativo nota come containers.
```
Comment (Fale) [inline content/it/docs/reference/glossary/docker.md:7]: s/containers/container/

File: content/it/docs/reference/glossary/docker.md [hunk 26]
```diff
@@ -0,0 +1,17 @@
...
+
+aka:
+tags:
+- fundamental
+---
+Docker (nelo specifico, Docker Engine) è una technologia software che offre una virtualizzazione a livello del sistema operativo nota come {{< glossary_tooltip text="containers" term_id="container" >}}.
```
Comment (Fale) [inline content/it/docs/reference/glossary/docker.md:13]: s/nelo/nello/
s/containers/container/

File: content/it/docs/reference/glossary/docker.md [hunk 27]
```diff
@@ -0,0 +1,17 @@
...
+
+<!--more-->
+
+Docker utilizza delle funzionalità di isolamente del kernel Linux come cgroups e kernele namespaces e un file system union-capable come OverlayFS e altro permettendo a container indipendenti di girare all'interno di una singola istanza Linux, eliminando il sovraccarico nell'avviare e manutenere delle virtual machines (VMs).
```
Comment (Fale) [inline content/it/docs/reference/glossary/docker.md:17]: s/kernele/kernel/
I think that we could translate "virtual machines", since "macchine virtuali" is a very common term afaics

File: content/it/docs/reference/glossary/kubeadm.md [hunk 28]
```diff
@@ -0,0 +1,19 @@
...
+aka: 
+tags:
+- tool
+- operation
+---
+ Un tool per installare velocemente Kubernetes e avviare un cluster sicuro.
+
+<!--more--> 
+
+Tu puoi usare kubeadm per installare sia il control plane che il {{< glossary_tooltip text="worker node" term_id="node" >}}.
```
Comment (Fale) [inline content/it/docs/reference/glossary/kubeadm.md:18]: I would remove the "Tu", since feels redundant.
The "control plane" has been considered female in Italian in other translation in previous pages and in other pages of this PR as well.

File: content/it/docs/reference/glossary/label.md [hunk 29]
```diff
@@ -0,0 +1,18 @@
...
+
+aka: 
+tags:
+- fundamental
+---
+ Tags di oggetti con attributi identificativi che sono significativi e pertinenti per gli utenti.
+
+<!--more--> 
+
+Labels sono delle coppie key/value che sono collegat a oggetti come {{< glossary_tooltip text="Pods" term_id="pod" >}}. Loro sono usate per organizzare e selezionare un sottoinsieme di oggetti.
```
Comment (Fale) [inline content/it/docs/reference/glossary/label.md:17]: s/Labels/Le label/
s/collegat/collegate/
s/Pods/Pod/
s/Loro/Esse/ (or "Le label")

File: content/it/docs/reference/glossary/master.md [hunk 30]
```diff
@@ -0,0 +1,15 @@
...
+id: master
+date: 2020-04-16
+short_description: >
+  Termine vecchio, usato come sinonimo per i nodi che ospitano la control plane.
+
+aka:
+tags:
+- fundamental
+---
+ Termine vecchio, usato come sinonimo per {{< glossary_tooltip text="nodes" term_id="node" >}} che ospitano la {{< glossary_tooltip text="control plane" term_id="control-plane" >}}.
```
Comment (Fale) [inline content/it/docs/reference/glossary/master.md:12]: s/nodes/node/

File: content/it/docs/reference/glossary/master.md [hunk 31]
```diff
@@ -0,0 +1,15 @@
...
+
+<!--more-->
+Il termine è ancora usato da alcuni strumenti di provisioning, come {{< glossary_tooltip text="kubeadm" term_id="kubeadm" >}}, e servizi gestiti, per mettere la {{< glossary_tooltip text="label" term_id="label" >}} `kubernetes.io/role` ai {{< glossary_tooltip text="nodi" term_id="node" >}} per controllare il posizionamento dei {{< glossary_tooltip text="pods" term_id="pod" >}} della {{< glossary_tooltip text="control plane" term_id="control-plane" >}} .
```
Comment (Fale) [inline content/it/docs/reference/glossary/master.md:15]: In this PR sometimes "node(s)" gets translated to "nodo/i", sometimes it stays as "node". I think we should agree on either form and stick with it. Is node a specific technical term or nor? This is the question :-D. I have no strong feeling toward either form.

File: content/it/docs/reference/glossary/pod.md [hunk 32]
```diff
@@ -0,0 +1,19 @@
...
+aka: 
+tags:
+- core-object
+- fundamental
+---
+ Il più piccolo e semplice oggetto in Kubernetes. Un pod rappresenta un gruppo di {{< glossary_tooltip text="container" term_id="container" >}} nel tuo cluster.
+
+<!--more--> 
+
+Un pod è tipicamente progettato per eseguire un singolo container primario. Può opzionalmente eseguire sidecar container che aggiungono funzionalità supplementari come logging. I Pod sono generalmetne gestiti da un {{< glossary_tooltip term_id="deployment" >}}.
```
Comment (Fale) [inline content/it/docs/reference/glossary/pod.md:18]: I think "pod" should not be capitalized

File: content/it/docs/reference/glossary/statefulset.md [hunk 33]
```diff
@@ -0,0 +1,22 @@
...
+- fundamental
+- core-object
+- workload
+- storage
+---
+ Gestisce deployment e la scalabilità di un gruppo di {{< glossary_tooltip text="Pods" term_id="pod" >}}, *e garantisce il corretto ordine e unicità* di questi Pods.
+
+<!--more--> 
+
+Come un {{< glossary_tooltip term_id="deployment" >}}, uno StatefulSet gestisce Pod che sono basati sulla stessa specifica di container. Contrariamente da un Deployment, uno StatefulSet mantiente una specifica identita per ogni Pod. Questi pod sono creati dalla stessa specifica, ma  non sono intercambiabili&#58; ogni pod a un identificativo persistente che si mantiene attraverso ogni rischedulazione.
```
Comment (Fale) [inline content/it/docs/reference/glossary/statefulset.md:20]: I would substitute "&#58;" with its symbol ":"

File: content/it/docs/reference/glossary/statefulset.md [hunk 34]
```diff
@@ -0,0 +1,22 @@
...
+
+Se tu vuoi usare un volume dello storage per avere la persistenza per il tuo carico di lavoro, tu puoi usare uno StatefulSet come parte della tua soluzione. Anche se i singoli Pod in uno StatefulSet sono suscettibili al fallimento, l'identificativo persistente del Pod rende semplice il collegamento dei volumi esistenti ai nuovi Pods che sostituiscono quelli falliti.
```
Comment (Fale) [inline content/it/docs/reference/glossary/statefulset.md:22]: I would remove both "tu", they feel unnecessary to me.

**Comment by fabriziopandini** [comment]
it seems there is something not properly set up in your CLA.
if this does not get fixed when you push next commit, I can help you to investigate what's going on

File: content/it/docs/concepts/overview/components.md (hunk 4 above)
Comment (Fale) [reply content/it/docs/concepts/overview/components.md:62]: Ok for me. In this case, s/control loops/control loop/ ;-)

File: content/it/docs/concepts/overview/components.md (hunk 1 above)
Comment (mbuccarello) [reply content/it/docs/concepts/overview/components.md:58]: I started with "la" but I prefered to use "il" for object and I think this should be used as rule.

File: content/it/docs/concepts/overview/components.md (hunk 2 above)
Comment (mbuccarello) [reply content/it/docs/concepts/overview/components.md:59]: Think about this sentence "Se hai una installazione CENTOS  on premise", I honestly prefer without "di"

File: content/it/docs/concepts/overview/components.md (hunk 7 above)
Comment (mbuccarello) [reply content/it/docs/concepts/overview/components.md:69]: If you are a nteworking guy, "rotte" is a popular term, but I think we should think about to use "route" as term.

File: content/it/docs/concepts/overview/components.md (hunk 8 above)
Comment (mbuccarello) [reply content/it/docs/concepts/overview/components.md:74]: I have no idea, because they removed the term worker node, alla the things are node, and there is no distinction for node componentes and node : )

File: content/it/docs/reference/glossary/master.md (hunk 31 above)
Comment (mbuccarello) [reply content/it/docs/reference/glossary/master.md:15]: I agree

File: content/it/docs/concepts/overview/components.md [hunk 35]
```diff
@@ -23,9 +23,9 @@ Questo è un diagramma di un cluster Kubernetes con tutti i componenti e le loro
 {{% capture body %}}
 ## Componenti della Control Plane
 
-La Control Plane è responsabile di tutte le decisioni globali sul cluster (ad esempio, lo scheduling), e l'individuazione e la risposta ad eventi derivanti dal cluster (ad esempio, l'avvio di un nuovo {{< glossary_tooltip text="pod" term_id="pod">}} quando il valore `replicas` di un deployment non è soddisfatto).
+I componenti della Control Plane sono responsabili di tutte le decisioni globali sul cluster (ad esempio, lo scheduling) oltre a rilevare e rispondere agli eventi del cluster (ad esempio, l'avvio di un nuovo {{< glossary_tooltip text="pod" term_id="pod">}} quando il valore `replicas` di un deployment non è soddisfatto).
```
[the remark this answers is not in the document]
Comment (mbuccarello) [reply content/it/docs/concepts/overview/components.md:26]: From my side it's ok your suggestions.

**Comment by mbuccarello** [comment]
Thanks guys for these review, let's catch up in these days to understand what's wrong with my CLA and push the changes suggested.

File: content/it/docs/reference/glossary/pod.md (hunk 32 above)
Comment (fabriziopandini) [reply content/it/docs/reference/glossary/pod.md:18]: All the API objects (Pod, Node, Service, Deployment) should be written in CamelCase (same as in the API)

**Comment by mbuccarello** [comment]
I signed it

File: content/it/docs/concepts/overview/components.md (hunk 7 above)
Comment (mbuccarello) [reply content/it/docs/concepts/overview/components.md:69]: I will allign rotte like the suggestion Fabrizio did with "network routes"

File: content/it/docs/concepts/overview/components.md (hunk 9 above)
Comment (mbuccarello) [reply content/it/docs/concepts/overview/components.md:90]: my fault resolved

File: content/it/docs/concepts/overview/components.md (hunk 10 above)
Comment (mbuccarello) [reply content/it/docs/concepts/overview/components.md:94]: my fault resolved

File: content/it/docs/reference/glossary/cluster.md (hunk 15 above)
Comment (mbuccarello) [reply content/it/docs/reference/glossary/cluster.md:14]: duplicated comment

File: content/it/docs/concepts/overview/components.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (mbuccarello) [reply content/it/docs/concepts/overview/components.md:69]: we follow your suggestion without plural meaning

File: content/it/docs/concepts/overview/components.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (mbuccarello) [reply content/it/docs/concepts/overview/components.md:74]: I don't agree, based on the english documentation is "Node Components".

**Comment by mbuccarello** [comment]
I signed it

File: content/it/docs/concepts/overview/components.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (fabriziopandini) [reply content/it/docs/concepts/overview/components.md:74]: It seems a little bit awkward to me in Italian, this is why I suggested drifting from the English translation.
However this is detail and I don't want to block for this

**Comment by fabriziopandini** [comment]
@sftim have you got some good suggestions to help @mbuccarello to unblock the CLA issue? I'm short of tips...

**Comment by sftim** [comment]
@mbuccarello you need to check that the email address for the commit in this PR matches the email address that you used when signing the CLA.

Do you believe that these match?
